### PR TITLE
Issue #242 Less verbose logging

### PR DESCRIPTION
**Pull request by gregw**
Reduced the verbosity of logs in VmApiProxyDelegate for #242
Also reviewed log levels in AppEngineAuthentication (and fixed some javadoc errors).

### CODE REVIEW

File: appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java [hunk 1]
```diff
@@ -161,7 +161,7 @@ public VmApiProxyDelegate() {
     try {
       byte responseData[] = runSyncCall(environment, packageName, methodName, requestData, timeoutMs);
       long end = System.currentTimeMillis();
-      logger.log(Level.INFO, String.format(
+      logger.log(Level.FINE, String.format(
```
Comment (aozarov) [inline appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java]: Do you think it is worth a `isLoggable` check?

File: jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/AppEngineAuthentication.java [hunk 2]
```diff
@@ -354,15 +354,14 @@ public void setIdentityService(IdentityService identityService) {
     }
 
     /**
-     * Validate a user identity. Validate that a UserIdentity previously created by a call to
-     * {@link #login(String, Object)} is still valid.
+     * Validate a user identity. Validate that a UserIdentity is still valid.
```
Comment (aozarov) [inline jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/AppEngineAuthentication.java]: 1) I think "Validate that a UserIdentity is still valid." should have been enough.
2) I think such doc change should have been applied on `LoginService` (as it does not seem to be specific to this impl).
3) I would normally leave out JavaDoc for methods that I override/implement unless they provide specifics for the implementation. In this case we just throw `UnsupportedException`

**Comment by aozarov** [comment]
Looks good. 2 nits.

File: appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java (hunk 1 above)
Comment (gregw) [inline appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java]: indeed, will do

File: jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/AppEngineAuthentication.java (hunk 2 above)
Comment (gregw) [inline jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/AppEngineAuthentication.java]: all good points.  I was just doing minimal edits to remove the hard javadoc errors of broken links.

**Comment by gregw** [comment]
denitted

File: appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java [hunk 3]
```diff
@@ -161,11 +161,13 @@ public VmApiProxyDelegate() {
     try {
       byte responseData[] = runSyncCall(environment, packageName, methodName, requestData, timeoutMs);
       long end = System.currentTimeMillis();
-      logger.log(Level.INFO, String.format(
+      if (logger.isLoggable(Level.FINE)) {
+        logger.log(Level.FINE, String.format(
           "Service bridge API call to package: %s, call: %s, of size: %s " +
-          "complete. Service bridge status code: %s; response " +
-          "content-length: %s. Took %s ms.", packageName, methodName, requestData.length, 200,
+            "complete. Service bridge status code: %s; response " +
```
Comment (aozarov) [inline appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmApiProxyDelegate.java]: No need to change as it is consistent with the file. So comment is more of an FYI about the line wrapping which should come before the '+' (https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break). I am surprised that the formatter didn't fix that for you.

**Comment by aozarov** [comment]
BTW, any preference of CI Travis vs CircleCi ?

**Comment by gregw** [comment]
@aozarov I'll defer to @joakime for a more definitive answer.     We've mostly used Jenkins and it is OK, but slightly confusing UI and I've not yet seen it integrated with github.    

I have used circle a little bit and have mixed feelings about it - it can be nice and simple, but if it does not work then it can be hard to configure.   It can also have problems with large builds and reviewing the output of them can lock up your browser.

No recent experience with travis.

**Pull request merged**